megaman/embedding/isomap.py

```diff
@@ -149,10 +147,10 @@ class Isomap(BaseEmbedding):
     random_state : numpy.RandomState or int, optional
         The generator or seed used to determine the starting vector for arpack
         iterations.  Defaults to numpy.random.RandomState
-    eigen_tol : float, optional. Tolerance for 'arpack' solver.
     path_method : string, optionl. method for computing graph shortest path.
         One of ['auto', 'D', 'FW', 'BF', 'J'].
         See `scipy.sparse.csgraph.shortest_path` for more information.
+    solver_kwds : any additional keyword arguments to pass to the selected eigen_solver
```

Comment:
Might it make sense to put links to the documentation of each method and
the keywords they can adjust in the Notes section of the eigendecomp module
documentation and then link people there?

On Mon, Mar 21, 2016 at 3:04 PM, Jake Vanderplas <notifications@github.com>
wrote:

> In megaman/embedding/isomap.py
> <https://github.com/mmp2/megaman/pull/51#discussion_r56905908>:
>
> >      path_method : string, optionl. method for computing graph shortest path.
> >          One of ['auto', 'D', 'FW', 'BF', 'J'].
> >          See `scipy.sparse.csgraph.shortest_path` for more information.
> > +    solver_kwds : any additional keyword arguments to pass to the selected eigen_solver
>
> Could be good to list solvers and examples of keywords they accept (if
> it's too long, maybe put it under Notes below and reference that here)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mmp2/megaman/pull/51/files/834527a478b0349417a2fbb9880092df310ba0e6#r56905908>
>